src/components/Main/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
```

Comment:
html이 시맨틱하게 사용이 되면 더 좋을 것 같네요!
```suggestion
const Container = styled.footer`
```